entproto/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM golang:1.17.3
```

Comment:
Why not `latest`? 

```suggestion
FROM golang
```

Reply:
I prefer being specific to make this as deterministic as i can.